package.json

```diff
@@ -25,36 +24,37 @@
     "travis:lint": "npm run lint && npm run security",
     "travis:test": "npm run test -- --runInBand",
     "travis:coverage": "npm run test:coverage -- --runInBand",
+    "appveyor:test": "npm run test",
     "webpack-defaults": "webpack-defaults"
   },
   "dependencies": {
-    "async": "2.4.1",
-    "webpack-sources": "^1.0.1"
+    "async": "^2.5.0",
+    "schema-utils": "^0.3.0",
+    "webpack-sources": "^1.0.0"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-jest": "^20.0.3",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-env": "^1.5.1",
-    "cross-env": "^4.0.0",
-    "del-cli": "^0.2.1",
-    "eslint": "^3.19.0",
-    "eslint-config-webpack": "^1.2.1",
-    "eslint-plugin-import": "^2.2.0",
-    "jest": "^20.0.4",
-    "lint-staged": "^3.4.0",
-    "nodemon": "^1.11.0",
-    "nsp": "^2.6.3",
-    "pre-commit": "^1.2.2",
+    "babel-cli": "^6.0.0",
+    "babel-jest": "^21.0.0",
+    "babel-plugin-transform-object-rest-spread": "^6.0.0",
+    "babel-polyfill": "^6.0.0",
+    "babel-preset-env": "^1.0.0",
+    "cross-env": "^5.0.0",
+    "del-cli": "^1.0.0",
+    "eslint": "^4.0.0",
+    "eslint-config-webpack": "^1.0.0",
+    "eslint-plugin-import": "^2.0.0",
+    "jest": "^21.0.0",
+    "lint-staged": "^4.0.0",
+    "nsp": "^2.0.0",
+    "pre-commit": "^1.0.0",
     "standard-version": "^4.0.0",
-    "webpack-defaults": "^1.4.0"
+    "webpack-defaults": "^1.0.0"
```

Comment:
Should be `"webpack-defaults": "^1.6.0"`

Reply:
When using `^` only the major version range counts, this will install the latest available in the semver minor/path ranges => `v1.6.0`

![package](https://user-images.githubusercontent.com/5419992/31106824-78d846ca-a7ee-11e7-9aab-45e905bcfe29.png)

Reply:
Yeah, that part I get. The other way is just an easy way to see which version it's running without having to go digging. Particularly given that most of our Minor releases aren't strictly backwards compatible